README.md

```diff
@@ -197,6 +197,19 @@ Key:
 * `-` The Kubernetes version has features or API objects that Volcano can't use.
 
 
+## Citing Volcano
+
+If Volcano helps your research, we appreciate your citations. Here is the BibTeX entry:
+
+```bibtex
+@misc{volcano2025,
+  title={Volcano: A Cloud Native Batch System},
+  author={Ma, Klaus and Wang, Kevin and Li, Zenlin and others},
```

Comment:
Should be Klaus Ma, Kevin Wang

Reply:
sure. i'll update it. shall i keep `and others`?

Reply:
i researched on it and it seems that bibtex parser(s) expect `and` between different names.

Reply:
done.